tools/worker-runner/perms/perms_windows.go

```diff
@@ -160,12 +160,7 @@ func verifyPrivateToOwner(filename string) (err error) {
 
 // Get the SID of the current user
 func getCurrentUser() (sid *windows.SID, err error) {
-	token, err := windows.OpenCurrentProcessToken()
-	if err != nil {
-		return
-	}
-	defer token.Close()
-
+	token := windows.GetCurrentProcessToken()
 	tokenuser, err := token.GetTokenUser()
```

Comment:
Oh - this isn't something new you've added, but won't this `err` shadow the one that gets returned?

Reply:
Addressed in https://github.com/taskcluster/taskcluster/pull/6829 :) Nice find!